### PR TITLE
Adding FAB for creating a new trip.

### PR DESCRIPTION
**Pull request by adikakade31**
Added FAB for creating a new trip.
Using google search for adding places

### CODE REVIEW

File: app/src/main/res/layout/fragment_new_trip.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"/>
+
+    <Button
+        android:id="@+id/btStartNewTrip"
+        android:text="@string/start_new_trip"
+        android:layout_below="@+id/tvDestination"
+        android:layout_marginTop="10dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>s
```
Comment (heyhuyen) [inline app/src/main/res/layout/fragment_new_trip.xml:32]: looks like there is a trailing "s" at the end of this line

**Comment by heyhuyen** [comment]
looks great!

**Review by heyhuyen — APPROVED** [review]

File: app/src/main/res/layout/fragment_new_trip.xml (hunk 1 above)
Comment (adikakade31) [reply app/src/main/res/layout/fragment_new_trip.xml:32]: oh! please remove that :)

Comment (heyhuyen) [reply app/src/main/res/layout/fragment_new_trip.xml:32]: Okay I'm gonna merge this and then I'll merge mine and remove the extra "s".

**Pull request merged**